src/service/xhr-impl.js

```diff
@@ -191,8 +193,8 @@ export class Xhr {
       init.headers['Content-Type'] = 'application/json;charset=utf-8';
```

Comment:
Whoops, I thought the second param was for `Content-Type`. Thanks for looking into it.